kafka-ui-api/src/main/java/com/provectus/kafka/ui/serdes/builtin/ProtobufRawSerde.java

```diff
@@ -0,0 +1,59 @@
+package com.provectus.kafka.ui.serdes.builtin;
+
+import com.google.protobuf.UnknownFieldSet;
+import com.provectus.kafka.ui.exception.ValidationException;
+import com.provectus.kafka.ui.serde.api.DeserializeResult;
+import com.provectus.kafka.ui.serde.api.RecordHeaders;
+import com.provectus.kafka.ui.serde.api.SchemaDescription;
+import com.provectus.kafka.ui.serdes.BuiltInSerde;
+import java.util.Map;
+import java.util.Optional;
+import lombok.SneakyThrows;
+
+public class ProtobufRawSerde implements BuiltInSerde {
+
+  public static String name() {
+    return "ProtobufRaw";
+  }
+
+  @Override
+  public Optional<String> getDescription() {
+    return Optional.empty();
+  }
+
+  @Override
+  public Optional<SchemaDescription> getSchema(String topic, Target type) {
+    return Optional.empty();
+  }
+
+  @Override
+  public boolean canSerialize(String topic, Target type) {
+    return false;
+  }
+
+  @Override
+  public boolean canDeserialize(String topic, Target type) {
+    return true;
+  }
+
+  @Override
+  public Serializer serializer(String topic, Target type) {
+    throw new UnsupportedOperationException();
+  }
+
+  @Override
+  public Deserializer deserializer(String topic, Target type) {
+    return new Deserializer() {
+        @SneakyThrows
+        @Override
+        public DeserializeResult deserialize(RecordHeaders headers, byte[] data) {
+            try {
+              UnknownFieldSet unknownFields = UnknownFieldSet.parseFrom(data);
+              return new DeserializeResult(unknownFields.toString(), DeserializeResult.Type.STRING, Map.of());
```

Comment:
I think we should return json representation of UnknownFieldSet, not just string. 
Potential format can be: 
```
{ 
 "field_1": 234,
 "field_2": "str",
 "field_3":  [ { "field_1": 5555, .... } ] 
}
```
This will allow us to apply smart filters to such messages. You can check UnknownFieldSet.toString() for implementation inspiration

Reply:
The reason I chose the text format for this is because it is [well documented](https://protobuf.dev/reference/protobuf/textformat-spec/) and does not require a bespoke format. Unknown fields are not supported in the official JsonFormat class. For smart filters, it will be necessary to use a protobuf file and the ProtobufFile serde. I hope you agree.

Reply:
well, I agree in general, but still, I think smart filter usage is more important here... @Haarolean what do you think?

Reply:
discssed internally with Haarolean  - lets merge it as is and wait until some user request (if any)

Reply:
Thanks. Apologies I didn't get to it for this PR. If such a request comes along, feel free to tag me on it and I'll do it then.